pan-domain-auth-verification/src/main/scala/com/gu/pandomainauth/service/CookieUtils.scala

```diff
@@ -23,7 +24,7 @@ object CookieUtils {
       authUser.user.avatarUrl.map(a => s"&avatarUrl=$a").getOrElse("") +
       s"&system=${authUser.authenticatingSystem}" +
       s"&authedIn=${authUser.authenticatedIn.mkString(",")}" +
-      s"&expires=${authUser.expires}" +
+      s"&expires=${authUser.expires.toEpochMilli}" +
```

Comment:
As we've established, the `expires` field in the cookie payload is in **milliseconds**:

> firstName=Roberto&lastName=Tyley&...&**expires=1742386198000**&multifactor=true